### ISSUE TITLE
Ability to inactivate animated gifs in avatars

### ISSUE THREAD

**Issue by davidbgk** [elk-zone/elk#830]
It can quickly be annoying/disturbing to have a timeline full of micro-mouvements.

This option exists in the default web client and it can be important for accessibility to some people:

<img width="627" alt="Capture d’écran, le 2023-01-06 à 11 44 12" src="https://user-images.githubusercontent.com/3556/211057784-5337853b-f392-4e5f-84e6-c9e7f33c6e30.png">

**Comment by Geobert** [comment]
I don’t know if it is related to this issue, but we can have animated username too
https://main.elk.zone/mamot.fr/@mttaggart@fosstodon.org

(blinking cursor in the name)

**Comment by boehs** [comment]
> (blinking cursor in the name)

This is (I think) an emoji, which would instead be *hey, can we disable emoji animations*, imo totally valid.

**Comment by prazdevs** [comment]
As part of a11y, any animation should be disabled on reduced-motion and as a setting as well. I'm even surprised autoplaying emojis can be in names too 🤔

**Comment by shuuji3** [comment]
Although it has not been documented (https://docs.joinmastodon.org/methods/preferences/#get), the `autoplay:gifs` preference was actually available since Mastodon v4.1.0 (released on Feb 2023): https://github.com/mastodon/mastodon/releases/tag/v4.1.0

**Example response from `/api/v1/preferences`**

```json
{
  "posting:default:visibility": "public",
  "posting:default:sensitive":false,
  "posting:default:language":"en",
  "reading:expand:media":"default",
  "reading:expand:spoilers":false,
  "reading:autoplay:gifs":false
}
```

Note that the latest version of Elk already uses the `prefer-reduced-motion` for both animated avatar and emoji.
